app/models/childcare_envelope.rb

```diff
@@ -0,0 +1,13 @@
+class ChildcareEnvelope < ActiveRecord::Base
+  belongs_to :child
+  belongs_to :recipient, class_name: 'Attendee'
```

Comment:
If we're now going to send the envelope to all parents then this `belongs_to :recipient` association might not make sense anymore. Instead it might make sense to remove this association and just rely on the association between the child and family, then the family would `have_many :childcare_envelopes, through: :children`. What do you think?